adapter/src/index.js

```diff
@@ -15,6 +16,7 @@ const App = ({ url, apiVersion, appName, children }) => (
                 <HeaderBar appName={appName} />
                 <AuthBoundary url={url}>
                     <div className="app-shell-app">{children}</div>
+                    <Alerts />
```

Comment:
I felt that inserting `Alerts` here, as a child of the `AuthBoundary` and sibling of  `div.app-shell-app` was most appropriate. But that did force me to change prop-types on the `AuthBoundary`.